pkg/cvo/status.go

```diff
@@ -354,6 +360,35 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 	return err
 }
 
+func setTaintedCapabilitiesCondition(config *configv1.ClusterVersion, tainted []configv1.ClusterVersionCapability,
+	now metav1.Time) {
+
+	if len(tainted) > 0 {
+		message := "The following capabilities could not be disabled: "
+		caps := make([]string, len(tainted))
+		for i, c := range tainted {
+			caps[i] = string(c)
+		}
+		message = message + strings.Join([]string(caps), ", ")
+
+		resourcemerge.SetOperatorStatusCondition(&config.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+			Type:               TaintedCapabilities,
+			Status:             configv1.ConditionTrue,
+			Reason:             "Attempt to disable previousily enabled capability",
```

Comment:
`Reason` should be [a CamelCase slug][1].  Maybe `CapabilitiesImplicitlyEnabled` or whatever, to use @bparees' preferred words (assembled into a camel CaseSlug by me).  Maybe just echo the `Type`?  I don't think we're planning on putting in the effort to distinguish the various cases for implicitly enabling caps.

[1]: https://github.com/openshift/api/blob/67c28690af52a69e0b8fa565916fe1b9b7f52f10/config/v1/types_cluster_operator.go#L131